test/new-e2e/tests/agent-shared-components/forwarder/nss_failover_test.go

```diff
@@ -159,10 +159,14 @@ func (v *multiFakeIntakeSuite) TestNSSFailover() {
 	setHostEntry(v.T(), v.Env().Host, intakeName, fakeintake1IP)
 
 	// configure agent to use the custom intake, set connection_reset_interval, use logs, and processes
+	scheme := "http"
+	if strings.HasPrefix(v.Env().Fakeintake1.URL, "https://") {
+		scheme = "https"
+	}
```

Comment:
Can we be sure that both fakeintakes have the same scheme ? This test assumes so...